### ISSUE TITLE
App forces/takes focus periodically

### ISSUE THREAD

**Issue by squirtbrnr** [home-assistant/iOS#1486]
**Device model, version and app version**
Model Name: MacBook Pro (Retina, 13-inch, Early 2015)
macOS Version: macOS Catalina, 10.15.7
App Version: Mac App Store - 2021.2.2 (2021.55)
Safari Version: 14.0.3 (15610.4.3.1.6, 15610)

**Home Assistant Core Version**
2021.2.3

**Describe the bug**
periodically while using my Mac the front most app in focus will switch from whatever app I am using to the HA Companion app.

**To Reproduce**
Open HA Companion App.  Close the window (red circle).  Do not quit the app, leave the app running in the background.  Use any other app for a period of time and watch the menu bar switch focus to the HA Companion App.

**Expected behavior**
The app should be able to run in the background without forcing focus away from the front most app.

**Screenshots**

**Additional context**
My App is set to update sensors periodically every 5 minutes, but I have not timed if this app focus occurs in that same amount of time.  Previous versions of this app (before the latest version that added the front most app sensor) would either self-quit when closing the window and moving focus to another app, or the app would continue running in the background but would not take focus away from the front most app.

**Comment by jhass** [comment]
Why did you close this? I'm still seeing this with 2022.3

**Comment by squirtbrnr** [comment]
What macOS version are you running?  I only saw this happen with macOS Catalina.  When I upgraded to Monterey (was a fresh install, not an in-place upgrade) I have not had this issue happen again.  The only time it has happened is when I purposefully disconnected from the internet (turned off WiFi) and the app was running, but closed.  After a period of time, the app popped up saying it failed to connect, but if I closed it again, it would not come up again.
